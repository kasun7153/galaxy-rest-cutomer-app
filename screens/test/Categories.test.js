import React from "react"
import Categories from '../../screens/categories'
import { render, fireEvent, act,waitFor  } from '@testing-library/react-native';
import {Provider} from 'react-redux';
import configureStore from 'redux-mock-store';

const flushMicrotasksQueue = () =>
  new Promise((resolve) => setImmediate(resolve));

describe('<Categories />',  () => {
  const mockStore = configureStore([]);
  const push = jest.fn();
  it("renders default elements", async () => {

    const store = mockStore();
    const {getByTitle}=render(<Provider store={store}><Categories
      navigation={
        {
          setOptions: push
        }
      }/></Provider>);
    expect(fetch.mock.calls).toMatchSnapshot();
  });
})

