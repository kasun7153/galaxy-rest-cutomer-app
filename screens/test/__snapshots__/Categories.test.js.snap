// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Categories /> renders default elements 1`] = `Array []`;
